recipes/default.rb

```diff
@@ -71,11 +71,11 @@
 
 init_template_dir = value_for_platform_family(
   ["rhel", "fedora"] => "rhel",
-  "debian" => "debian"
+  ["debian", "raspbian"] => "debian"
```

Comment:
No, family is still 'debian' https://github.com/chef/ohai/blob/master/lib/ohai/plugins/linux/platform.rb#L114-L115